packages/design-system/src/components/AskAssistantChat/__tests__/AskAssistantChat.spec.ts

```diff
@@ -0,0 +1,131 @@
+import { render } from '@testing-library/vue';
+import AskAssistantChat from '../AskAssistantChat.vue';
+
+describe('AskAssistantChat', () => {
+	it('renders default placeholder chat correctly', () => {
+		const { container } = render(AskAssistantChat, {
+			props: {
+				user: { firstName: 'Kobi', lastName: 'Dog' },
+			},
+		});
+		expect(container).toMatchSnapshot();
+	});
+	it('renders chat with messages correctly', () => {
+		const { container } = render(AskAssistantChat, {
+			props: {
```

Comment:
Should we have a function that create this objects? By default would have this values and they can be overrides via parameters. That way we do not need to do so much repetition? 🤔

Reply:
I tried but since the assistant types are so tangled up this will require a lot of TypeScript gymnastics or a lot of `if/else` branches which will make util function pretty ugly so decided to keep it this way. Let me know what you think @RicardoE105